src/test/ui/consts/std/cell.rs

```diff
@@ -1,18 +1,26 @@
+#![feature(const_refs_to_cell)]
+
 use std::cell::*;
 
 // not ok, because this would create a silent constant with interior mutability.
 // the rules could be relaxed in the future
 static FOO: Wrap<*mut u32> = Wrap(Cell::new(42).as_ptr());
-//~^ ERROR cannot borrow a constant which may contain interior mutability
+//~^ ERROR encountered dangling pointer
+const FOO_CONST: Wrap<*mut u32> = Wrap(Cell::new(42).as_ptr());
+//~^ ERROR encountered dangling pointer
 
 static FOO3: Wrap<Cell<u32>> = Wrap(Cell::new(42));
+const FOO3_CONST: Wrap<Cell<u32>> = Wrap(Cell::new(42));
+
 // ok
 static FOO4: Wrap<*mut u32> = Wrap(FOO3.0.as_ptr());
+const FOO4_CONST: Wrap<*mut u32> = Wrap(FOO3_CONST.0.as_ptr());
+//~^ ERROR encountered dangling pointer
```

Comment:
The comment above says "ok" but there's an error; something is off here.

Reply:
The "ok" is for the `static`. I'll add a comment to each item explaining why it is ok or not

Reply:
Well, elsewhere in this file the comment is for the entire following "block" (separated by empty lines), so I used the same interpretation here. :)